sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/http/client/DefaultHttpClient.java

```diff
@@ -36,15 +39,20 @@
 import java.util.Base64;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
+import java.util.regex.Pattern;
 
 /**
  * HttpClient implementation using {@link HttpURLConnection} to send requests and receive responses.
  */
 class DefaultHttpClient implements HttpClient {
     private static final ClientLogger LOGGER = new ClientLogger(DefaultHttpClient.class);
+    public static final String LAST_EVENT_ID = "Last-Event-Id";
     private final long connectionTimeout;
     private final long readTimeout;
     private final ProxyOptions proxyOptions;
+    private static final String DEFAULT_EVENT = "message";
+    private final Pattern DIGITS_ONLY = Pattern.compile("^[\\d]*$");
```

Comment:
These constants can probably be inside the ServerSentEvent type instead of the DefaultHttpClient.